usasearch_api/usasearch_api.module

```diff
@@ -201,3 +202,85 @@ function usasearch_api_update_doc_action($node, $context) {
 function usasearch_api_delete_doc_action($node, $context) {
   usasearch_api_node_delete($node);
 }
+
+/**
+ * Implements hook_module_implements_alter().
+ */
+function usasearch_api_module_implements_alter(&$implementations, $hook) {
+  if ($hook == 'cron') {
+    unset($implementations['search']);
+  }
+}
+
+/**
+ * implimentation of hook_form_alter()
+ */
+function usasearch_api_form_search_admin_settings_alter(&$form, &$form_state, $form_id){
+  $items = drupal_map_assoc(array(10, 20, 50, 100, 200, 500,1000,10000));
+  $form['indexing_throttle']['search_cron_limit'] = array(
+    '#type' => 'select',
+    '#title' => t('Number of items to index per cron run'),
+    '#default_value' => variable_get('search_cron_limit', 100),
+    '#options' => $items,
+    '#description' => t('The maximum number of items indexed in each pass of a <a href="@cron">cron maintenance task</a>. If necessary, reduce the number of items to prevent timeouts and memory errors while indexing.', array('@cron' => url('admin/reports/status')))
```

Comment:
`<a href="@cron">cron maintenance task</a>` should be replaced with @cron_link, which should be generated by the l() function.